test/uk/gov/hmrc/cataloguefrontend/FakeApplicationBuilder.scala

```diff
@@ -61,4 +72,18 @@ trait FakeApplicationBuilder
         ))
       .build()
 
+  def cookieForAuth(value: String): WSCookie = {
```

Comment:
wonder if this is something we could add to `internal-auth-client` to help others too (although we don't currently have an `internal-auth-client-test` module)